BUILD.md

```diff
@@ -258,13 +260,16 @@ It fixes some source files to add missing `#include`s.
 
     Single-config generators:
 
+    Pass the CMake variable [`CMAKE_BUILD_TYPE`][build_type]
+    and make sure it matches the `build_type` setting you chose in the previous
+    step.
```

Comment:
I'm not sure how nitpicky I want to be about this. If you build both configurations, as the example above shows, conan is smart enough to combine the outputs in the toolchain file, and you can successfully build with either value in `CMAKE_BUILD_TYPE`. In other words, you don't necessarily have to wipe the build directory in between to choose one conan config or the other. 

I think this is clear enough as-is, but consider:
```suggestion
    Pass the CMake variable [`CMAKE_BUILD_TYPE`][build_type]
    and make sure it matches the one of the `build_type` settings
    you chose in the previous step.
```